my-app/src/App.tsx

```diff
@@ -0,0 +1,52 @@
+import './App.css';
+
+const USERS: User[] = [
+  { name: "thanh", position: "$1", start_date: '12 Nov 2024' },
+  { name: "kim", position: "$1", start_date: '12 Nov 2024' },
+  { name: "nguyen", position: "$1", start_date: '12 Nov 2024' },
+  { name: "anh", position: "$1", start_date: '12 Nov 2024' }
+];
+
+interface User {
+  name: string;
+  position: string;
+  start_date: string;
+}
+
+const UserRow = ({ user }: { user: User }) => {
```

Comment:
Create interface for UserRow

```ts
interface UserRowProps {
  user: User
}
```
then use:
```
const UserRow = ({ user }: UserRowProps)
```

similar for UserList

Reply:
Best practice here is pass individual prop like `name`, `position`, `startDate` instead of pass the whole `user`

```tsx
const UserRow = ({ name, position, startDate }: UserRowProps)
```

Reply:
fixed